src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfSource = sourceString
+    .split(';')
+    .filter((part) => part.trim() !== '');
```

Comment:
The variable name 'arrayOfSource' is not descriptive of its contents. Consider renaming it to something more indicative of its purpose, such as 'styleDeclarations'.